.env.sample

```diff
@@ -0,0 +1,8 @@
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_USER=POSTGRES_USER
+POSTGRES_DB=POSTGRES_DB
+POSTGRES_PASSWORD=POSTGRES_PASSWORD
+SECRET_KEY=SECRET_KEY_OF_PR
+DEBUG=1
+DJANGO_ALLOWED_HOSTS=0.0.0.0
+POSTGRES_PORT=5432
```

Comment:
write here smth like that for all of variables - < your POSTGRES HOST> , <your POSTGRES USER> , etc.